mimic/test/test_maas.py

```diff
@@ -0,0 +1,677 @@
+        self.assertEquals(resp.code, 201)
+        return resp
+
+    def getXobjectIDfromResponse(self, resp):
+        xobjectid = None
+        for h in resp.headers.getAllRawHeaders():
+            if h[0].lower() == 'x-object-id':
+                xobjectid = h[1][0]
+                break
+        return xobjectid
+
+    def get_ecan_object_ids(self):
+        """
+        Get the Entity, check, alarm an notification(plan) objects created by setUp()
+        """
+        req = request(self, self.root, "GET", self.uri + '/views/overview', '')
+        resp = self.successResultOf(req)
+        self.assertEquals(resp.code, 200)
+        data = self.get_responsebody(resp)['values'][0]
+        entity_id = data['entity']['id']
+        check_id = data['checks'][0]['id']
+        alarm_id = data['alarms'][0]['id']
+        nt_id = None
+        np_id = None
+        sp_id = None
+
+        req = request(self, self.root, "GET", self.uri + '/notifications', '')
+        resp = self.successResultOf(req)
+        self.assertEquals(resp.code, 200)
+        data = self.get_responsebody(resp)
+        for nt in data['values']:
+            if nt['id'] != 'ntTechnicalContactsEmail':
+                nt_id = nt['id']
+
+        req = request(self, self.root, "GET", self.uri + '/notification_plans', '')
+        resp = self.successResultOf(req)
+        self.assertEquals(resp.code, 200)
+        data = self.get_responsebody(resp)
+        for np in data['values']:
+            if np['id'] != 'npTechnicalContactsEmail':
+                np_id = np['id']
+
+        req = request(self, self.root, "GET", self.uri + '/suppressions', '')
+        resp = self.successResultOf(req)
+        self.assertEquals(resp.code, 200)
+        data = self.get_responsebody(resp)
+        for sp in data['values']:
+            sp_id = sp['id']
+
+        return {'entity_id': entity_id, 'check_id': check_id, 'alarm_id': alarm_id,
+                'nt_id': nt_id, 'np_id': np_id, 'sp_id': sp_id}
+
+    def setUp(self):
+        """
+        Setup MaasApi helper object & uri 'n stuff
+        """
+        helper = APIMockHelper(self, [MaasApi(["ORD"])])
+        self.root = helper.root
+        self.uri = helper.uri
+        entity_id = None
+        check_id = None
+        alarm_id = None
+        nt_id = None
+        np_id = None
+        sp_id = None
+        entity_id = self.getXobjectIDfromResponse(self.createEntity('ItsAnEntity'))
+        check_id = self.getXobjectIDfromResponse(self.createCheck('ItsAcheck', entity_id))
+        alarm_id = self.getXobjectIDfromResponse(self.createAlarm('ItsAnAlarm', entity_id, check_id))
+        nt_id = self.getXobjectIDfromResponse(self.createNotification('ItsANotificationTarget'))
+        np_id = self.getXobjectIDfromResponse(self.createNotificationPlan('ItsANotificationPlan'))
+        sp_id = self.getXobjectIDfromResponse(self.createSuppression('ItsASuppression'))
+        self.assertNotEquals(None, entity_id)
```

Comment:
I'd also assert more about the returned value here: "obviously" it shouldn't be `None`, but what *should* it be?  If it's randomly generated, what format should it be in?  If it's supposed to be an instance of a particular type, you can use `assertIsInstance`?  If it's supposed to be a particular length, you can assert about its `len`.